<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortInfoWidget</class>
 <widget class="QWidget" name="PortInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_pLblPortIcon">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPortNr">
       <property name="text">
        <string>Portnumber:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pLblPortNr">
       <property name="text">
        <string notr="true">Nr</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblPortId">
       <property name="text">
        <string>Port Id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pLblPortId">
       <property name="text">
        <string notr="true">Id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="m_pLblUsbDevicePort">
       <property name="text">
        <string>Usb Device Jack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pLblUsbHostJackPort">
       <property name="text">
        <string>USB Host Jack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pLblEthernetSession">
       <property name="text">
        <string>Ethernet Jack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pLblJackSpecific">
       <property name="text">
        <string>Jack Specific</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_pLblPortName">
       <property name="text">
        <string>Port Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pTxtPortName">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="m_pCbInputEnabled">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Input Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pCbOutputEnabled">
       <property name="text">
        <string>Output Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
